lib/routes/articles.js

```diff
@@ -0,0 +1,30 @@
+const { Router } = require("express");
+const router = Router();
+
+const { readArticles, readArticle } = require("../models/articles");
+
+router.get("/", async (request, response) => {
+  try {
+    const articles = await readArticles();
+    if (!articles) {
+      response.status(404).send(`Articles not found`);
```

Comment:
Make sure to return in this if-statement. Otherwise, the following lines are executed too.
Happens multiple times.

```suggestion
      return response.status(404).send(`Articles not found`);
```

Reply:
Thanks!